include/NovelRT/LoggingService.h

```diff
@@ -34,6 +34,7 @@ namespace NovelRT {
     void logDebugLine(const std::string& message);
     void logInternal(const std::string& message, LogLevel level);
     void setLogLevel(LogLevel level);
+    void throwIfNullPtr(const void* const object, const std::string& exceptionMessage);
```

Comment:
Two `const`s here?

Reply:
Yes - an immutable pointer to an immutable object.

Reply:
:+1: